gmplot/gmplot.py

```diff
@@ -25,7 +26,10 @@ def __init__(self, center_lat, center_lng, zoom):
         self.heatmap_points = []
         self.radpoints = []
         self.gridsetting = None
-        self.coloricon = os.path.join(os.path.dirname(__file__), 'markers/%s.png')
+        if platform.system() == 'Windows':
+            self.coloricon = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'markers\\%s.png')
```

Comment:
The joiner was the issue; it seems to work without abspath.  However, the re.sub part is essential; I couldn't get it to work without that line in any major browser on windows.